workspace/__lib__/xod/math/max/patch.test.tsv

```diff
@@ -0,0 +1,5 @@
+IN1	IN2	OUT
+0.1	0.01	0.1
+-0.5	0.5	0.5
+500	2300	2300
+-20	-1	-1
```

Comment:
Tests with infinities and NaNs are desired much.

Reply:
Done!